.github/workflows/manual-deploy-dexynth-gateway.yml

```diff
@@ -130,10 +130,12 @@ jobs:
                -v /proc/:/host/proc/:ro \
                -v /opt/datadog-agent/run:/opt/datadog-agent/run:rw \
                -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro \
+               --log-opt max-file=3 --log-opt max-size=10m \
```

Comment:
what happens when we reach the limit? do we truncate the file?

why is it set to only 10 MB?

Reply:
it's just a rotating log file - will delete the oldest - so the total logs available will be the last 3*10MB (30mb). It's just what seemed useful originally as a set-point based on how we were using. we can definitely swap the params to different values though... @zkokelj